sympy/core/evalf.py

```diff
@@ -353,8 +353,7 @@ def get_integer_part(expr, no, options, return_ints=False):
     # positive or negative (which may fail if very close).
     def calc_part(re_im, nexpr):
         from sympy.core.add import Add
-        n, c, p, b = nexpr
-        is_int = (p == 0)
+        is_int = (nexpr[2] == 0)
```

Comment:
I feel that it is more readable to separate `nexpr` into its 4 components rather than take the third element without a name. Would a compromise between the two be better?
```python
exponent = nexpr[2]
is_int = exponent == 0
```

Reply:
Sure, this does look better indeed. Since the other 3 variables were never used, I thought removing them would be better so less space is taken up, but I think a little compromise on space is fine if it helps in readability a lot.